### PR TITLE
use standard component for players in rankings

### LINKED ISSUE #277: use standard table hero/player image element for rankings
Currently we use a custom element developed by @bguzryanto, but we now have a fairly standard hero/player TD element that we use on the rest of our tables.

### PR DESCRIPTION
**Pull request by howardchung**
fixes #277

### CODE REVIEW

File: src/components/Heroes/RankingTable.jsx [hunk 1]
```diff
@@ -1,30 +1,17 @@
...
-    />
-    <h3 className={style.RankingPersonaName}>
-      <Link to={`/players/${row.account_id}`}>
-        {row.personaname}
-      </Link>
-    </h3>
-    {row.solo_competitive_rank ? <span>{row.solo_competitive_rank} MMR</span> : ''}
-  </div>),
+  displayFn: (row, col, field) => {
+    const subtitle = <span>{row.solo_competitive_rank ? `${row.solo_competitive_rank} MMR` : ''}</span>;
```
Comment (howardchung) [inline src/components/Heroes/RankingTable.jsx]: localize MMR string?

Use TableHeroImage component directly?

**Comment by howardchung** [comment]
Not sure what you mean compared to.

It looks more like the elements for peers/pros now, since it also uses TableHeroImage.

**Comment by mike-robertson** [comment]
Sorry I misunderstood. I thought you meant the big blocks with hero images. I think we should use the Tabbar component for this and make those tabs interact with the URL so they are linkable and part of the history for the back button. What do you think? At the very least it should match the style.

**Comment by howardchung** [comment]
We can, although maybe not for initial release.

**Pull request merged**